plugins/CoreAdminHome/templates/home.twig

```diff
@@ -61,16 +55,12 @@
         <div class="row">
             {% if hasDonateForm %}
                 <div class="col s12 {% if hasPiwikBlog %}m6{% endif %}">
-                    <div piwik-content-block content-title="{{ 'CoreHome_SupportPiwik'|translate }}">
-                        <div piwik-widget-loader='{"module":"CoreHome","action":"getDonateForm","widget": "0"}'></div>
-                    </div>
+                    <div piwik-widget-loader='{"module":"CoreHome","action":"getDonateForm","widget": "0"}'></div>
                 </div>
             {% endif %}
             {% if hasPiwikBlog %}
                 <div class="col s12 {% if hasDonateForm %}m6{% endif %}">
-                    <div piwik-content-block content-title="Piwik.org Blog">
-                        <div piwik-widget-loader='{"module":"RssWidget","action":"rssPiwik"}'></div>
-                    </div>
+                    <div piwik-widget-loader='{"module":"RssWidget","action":"rssPiwik"}'></div>
```

Comment:
Here we can see the benefit of rendering widget headlines automatically now, we do not have to define the title twice, we automatically use the widget title defined in the widget